packages/serverpod/lib/src/database/expressions.dart

```diff
@@ -358,6 +358,22 @@ class Constant extends Expression {
   }
 }
 
+/// A relation between two tables.
+class TableRelation {
+  /// Column in the referencing table.
+  final Column referencedColumn;
```

Comment:
Change name to foreignColumn and column, since the context gives us the information we need.